src/datetimepicker.windows.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ * @flow strict-local
+ */
+'use strict';
+
+import {requireNativeComponent, StyleSheet} from 'react-native';
+import type {WindowsNativeProps, WindowsDatePickerChangeEvent} from './types';
+import * as React from 'react';
+
+const styles = StyleSheet.create({
+  rnDatePicker: {
+    height: 32,
+    width: 150,
+  },
+});
+
+const RNDateTimePickerWindows = requireNativeComponent('RNDateTimePickerWindows');
+
+export default class DateTimePickerWindows extends React.Component<WindowsNativeProps> {
+  static defaultProps: WindowsNativeProps = {
```

Comment:
"The CalendarDatePicker uses the correct calendar for your app's default language, or you can set the CalendarIdentifier property to use a specific calendar system" (https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.calendardatepicker?view=winrt-18362#globalization)

Reply:
how is this exposed to RN apps though? what property do they set on the js side that will set the XAML CalendarIdentifier property for them?

Reply:
You're right, we don't currently expose it; I migrated the supported set of properties from Core. 
Also, I meant that it will adjust automatically to the locale - is that not enough? I can see how someone might want to change eg. the date format, but countries only ever use one version of the calendar

Reply:
opened issue: https://github.com/microsoft/react-native-windows/issues/4791

Reply:
@rectified95 it's actually quite common in some countries to use two calendars, e.g. Japan with the Imperial  Calendar for government events and Gregorian for everyday affairs. Same with Israel with religious events with the Jewish Calendar and the Gregorian for secular events.

I'm fine with this going in without this support right now, but please open an issue to track adding a property to expose it what calendar to use. @chrisglein @harinikmsft FYI

Reply:
I think they have `locale` for this? It's iOS only but we can probably make it work for Windows. https://github.com/react-native-community/datetimepicker#locale-optional-ios-only

Reply:
@asklar I've created microsoft/react-native-windows#4879